### PR TITLE
Overwrite not being passed to HDF5 and .lammpstrj and some small stuff (doc strings, pep8)

### PR DESCRIPTION
**Pull request by ctk3b**
Quick question: there are several docstrings with single quotes lying around with single quotes, e.g.:
https://github.com/mdtraj/mdtraj/blob/master/mdtraj/core/topology.py#L947

Is that intentional or just go ahead and fix it if I find em?

### CODE REVIEW

File: mdtraj/core/topology.py [hunk 1]
```diff
@@ -641,7 +641,7 @@ def residues(self):
 
     @property
     def n_residues(self):
-        """Get the number of residues in the Topology"""
```
Comment (mpharrigan) [inline mdtraj/core/topology.py]: Why are you *removing* the triple quotes in this case?

Comment (ctk3b) [inline mdtraj/core/topology.py]: Woops sorry. Fixed.

Comment (mpharrigan) [inline mdtraj/core/topology.py]: Haha cool

**Comment by rmcgibbo** [comment]
Thanks.

**Pull request merged**